sdk/monitor/Azure.Monitor.Query/api/Azure.Monitor.Query.netstandard2.0.cs

```diff
@@ -59,6 +65,7 @@ public partial class LogsQueryResult
     {
         internal LogsQueryResult() { }
         public Azure.Monitor.Query.Models.LogsQueryResultTable PrimaryTable { get { throw null; } }
+        public System.Text.Json.JsonElement Statistics { get { throw null; } }
```

Comment:
Not a final API shape. Swagger would be updated but I wanted to have something to tests.